### PR TITLE
CONTRIBUTING.md Fixed typos. Removed redundancies

### PR DESCRIPTION
**Pull request by foohasmallya**
"All submissions, including submissions by project members, require review.
We use GitHub pull requests for this purpose. Consult GitHub Help for more
information on using pull requests" is repeated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,29 +7,23 @@ just a few small guidelines you need to follow.
...
-this simply gives us permission to use and redistribute your contributions as
+the CLA simply gives us permission to use and redistribute your contributions as
 part of the project. Head over to <https://cla.developers.google.com/> to see
 your current agreements on file or to sign a new one.
 
 You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
-## Code reviews
```
Comment (adammichaelwood) [inline CONTRIBUTING.md]: I'm not sure why you deleted this.

Comment (foohasmallya) [reply CONTRIBUTING.md]: Hey Adam. Thanks for taking the time to review this PR. I must've removed it by accident/must've thought it's redundant. I don't remember exactly why. I've fixed the issue by adding the heading back to the file.

Comment (foohasmallya) [reply CONTRIBUTING.md]: Hey @adammichaelwood Just following up on this

Comment (foohasmallya) [reply CONTRIBUTING.md]: Hey @adammichaelwood Let me know if you need help with anything else

**Pull request closed without merging**